sources/Algo/src/Algo.cpp

```diff
@@ -87,14 +87,23 @@ GeneratorDefinitionAlgorithm::GeneratorDefinitionAlgorithm(Generators& gens, Bus
 void
 GeneratorDefinitionAlgorithm::operator()(const NodePtr& node) {
   auto& node_generators = node->generators;
+
+  auto is_model_diagram = [](GeneratorDefinition::ModelType model, inputs::Generator generator) {
+    return (model == GeneratorDefinition::ModelType::DIAGRAM_PQ_SIGNALN || model == GeneratorDefinition::ModelType::REMOTE_DIAGRAM_PQ_SIGNALN || 
+            model == GeneratorDefinition::ModelType::PROP_DIAGRAM_PQ_SIGNALN || model == GeneratorDefinition::ModelType::WITH_IMPEDANCE_DIAGRAM_PQ_SIGNALN) && 
+            !isDiagramValid(generator);
+ };
```

Comment:
alinea is incorrect: please apply formatting

Reply:
Ok thanks